core/Tracker/Request.php

```diff
@@ -27,6 +28,10 @@
  */
 class Request
```

Comment:
Basically here are many requests to avoid a lot of `Common::getRequestVar()` calls. Especially `getCustomTimestamp` and `getIdSite` is called for each tracking request 10 times or more often. `getIdSite` does even trigger an event each time it is called so that makes it much faster.